### ISSUE TITLE
Keras.layers are not working

### ISSUE THREAD

**Issue by chaitugujjula00** [chaitugujjula00/Machine-Learning-Models#1]
#For this code, I am getting following error. 

import tensorflow as tf
from tensorflow import keras
model = keras.Sequential()

model.add(keras.layers.conv2D(32, kernel_size=(3,3),activation='relu',input_size=(128,128,3)))
model.add(keras.layers.MaxPooling2D(pool_size=(2,2)))

Clone the repo to your desktop and create your own branch.
Resolve the issue in the file 'Face Mask Detection/face_mask_detection.ipynb'

![Screenshot 2024-06-15 214655](https://github.com/chaitugujjula00/Machine-Learning-Models/assets/117301977/79709379-9772-4430-bdd3-2747e92a3b40)
